### DISCUSSION TITLE
3D Atlas Blender Wallpapers

### DISCUSSION THREAD

**Discussion by enz0g-ui** [Atlas-OS/Atlas#603] [Community Artwork]
![atlas 3d](https://user-images.githubusercontent.com/127509075/224328464-a74f611f-e672-4719-8118-ce772ce55459.png)

**Comment by enz0g-ui** [comment]
![4Tl45 3d](https://user-images.githubusercontent.com/127509075/224333546-320d886b-bf55-4b02-aa53-cafc4ee73162.png)

**Comment by enz0g-ui** [comment]
![4Tl45 3d](https://user-images.githubusercontent.com/127509075/224336861-bf196521-aebd-4d73-9fe4-5814b1b3777e.png)

**Comment by LumiFae** [comment]
These look really nice! Are you ok with sharing the project file or maybe just the export? Thanks!

**Reply by enz0g-ui** [reply]
no problems, just replace the .zip by .blend! upload a .blend is not supported
[atlas.zip](https://github.com/Atlas-OS/Atlas/files/10955871/atlas.zip)

**Reply by LumiFae** [reply]
Can you upload the .blend in a zip format then?

**Reply by enz0g-ui** [reply]
[atlas.zip](https://github.com/Atlas-OS/Atlas/files/10961860/atlas.zip)

**Reply by enz0g-ui** [reply]
here it is!

**Comment by enz0g-ui** [comment]
![atlas_111](https://user-images.githubusercontent.com/127509075/225017393-83c69767-6a76-4b10-b09a-3124e491d624.png)
![atlas_11](https://user-images.githubusercontent.com/127509075/225017603-3da4d70c-0d54-4915-b05e-93c77e6f7e5c.png)
![atlas_1111](https://user-images.githubusercontent.com/127509075/225017741-cb8ac5c3-d6a7-41ae-89b5-b2df8a8c9c6f.png)
